src/main/java/org/jboss/tools/intellij/openshift/utils/ConfigWatcher.java

```diff
@@ -38,7 +45,8 @@ public void run() {
                     Path path = (Path) event.context();
                     path = config.getParentFile().toPath().resolve(path);
                     if (path.equals(config.toPath())) {
-                        listener.onUpdate(this);
+                        Config config = ConfigHelper.loadKubeConfig();
```

Comment:
That causes the test to fail fo me: you may get an exception because the kube config file is not fully written and this the config watcher will not call the listener and even worse, the watcher will exit the loop

Reply:
@jeffmaury good pointer. What's the test that's reporting this?

Reply:
1) Log out
2) Relaunched IJ
3) Log in